doc/source/user/basics.types.rst

```diff
@@ -171,7 +171,7 @@ properties of the type, such as whether it is an integer::
 
     >>> d = np.dtype(int)
     >>> d
-    dtype('int32')
+    dtype('int64')
```

Comment:
This is platform-dependent.  Not sure if there is a marker for that, or refguide is only really supported in Linux and related OSes.

Reply:
This should probably be ignored here then with something like:

```diff
-    >>> d = np.dtype(int)
+    >>> d = np.dtype(int)  # doctest:+ELLIPSIS
     >>> d
-    dtype('int64')
+    dtype('...') # Machine dependent, int32 or int64

     >>> np.issubdtype(d, np.integer)
     True
```

Reply:
Another option would be to replace `int` with `float`, as the latter always maps to `float64`.

Reply:
@oveee can you try one of these solutions?

Reply:
Comment: `tools/refguide_check.py` add ELLIPSIS to all tests so that the flag is not necessary (it would be applied to the return line also, not at `d = np.dtype(int)`.